### DISCUSSION TITLE
Import of line items

### DISCUSSION THREAD

**Discussion by JohnMaverick** [YetiForceCompany/YetiForceCRM#16506] [How to] [answered]
I read https://github.com/YetiForceCompany/YetiForceCRM/issues/16504 
and regarding near that I have a question: How to import invoices from vtiger together with line items?

**Comment by apcloic** [comment, marked as answer]
@JohnMaverick 
You have to build a custom script.
#16503

**Comment by mariuszkrzaczkowski** [comment]
another option is the first versions, probably 1.x, had a built-in migration from vtiger and then you would have to update to the latest one
